src/main/java/com/booksaw/betterTeams/ConfigManager.java

```diff
@@ -102,26 +104,34 @@ public void updateFromDefaultSave(boolean log) {
 		}
 
 		List<String> changes = updateFileConfig(Main.plugin.getResource(resourceName));
+		int migratedKeys = updateVariables();
 
 		if (log) {
-			if (changes.isEmpty()) {
+			if (changes.isEmpty() && migratedKeys == 0) {
 				logger.info("[BetterTeams] File is up to date");
 			} else {
-				logger.info("[BetterTeams] ==================================================================");
-				logger.info("[BetterTeams] File is not updated, adding values under the following references:");
+				if (!changes.isEmpty()) {
+					logger.info("[BetterTeams] ==================================================================");
+					logger.info("[BetterTeams] File is not updated, adding values under the following references:");
+
+					for (String str : changes) {
+						logger.info("[BetterTeams] - " + str);
+					}
+				}
 
-				for (String str : changes) {
-					logger.info("[BetterTeams] - " + str);
+				if (migratedKeys != 0) {
+					logger.info("[BetterTeams] " + resourceName + " is using legacy variables. Migration taking place.");
```

Comment:
Can this logging be moved to the updateVariables method as both the warning that migration is taking place, and the success message are given at the same time.

Reply:
Sure. I was just trying to follow your style with what you did with the updateFileConfig method.